plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/AppEngineLibrariesInPluginXmlTest.java

```diff
@@ -41,54 +41,124 @@
 public class AppEngineLibrariesInPluginXmlTest {
 
   private static final String APP_ENGINE_API_LIBRARY_ID = "appengine-api";
+  private static final String APP_ENGINE_ENDPOINTS_LIBRARY_ID = "appengine-endpoints";
+  private static final String OBJECTIFY_LIBRARY_ID = "objectify";
 
   @Test
-  public void testAppEngineAPIConfig() throws URISyntaxException, LibraryFactoryException {
+  public void testThereAreOnlyThreeLibraries() throws Exception {
     IConfigurationElement[] configurationElements =
         RegistryFactory.getRegistry()
           .getConfigurationElementsFor(AppEngineLibraryContainerInitializer.LIBRARIES_EXTENSION_POINT);
-    for (IConfigurationElement configurationElement : configurationElements) {
-      if (configurationElement.getAttribute("id").equals(APP_ENGINE_API_LIBRARY_ID)) {
-        Library appEngineLibrary = new LibraryFactory().create(configurationElement);
-        verifyAppEngineLibrary(appEngineLibrary);
-        return;
-      }
-    }
-    fail("App Engine API library with ID " + APP_ENGINE_API_LIBRARY_ID + " was not found in the plugin.xml");
+    assertThat(configurationElements.length, is(3));
+  }
+
+  @Test
+  public void testAppEngineAPIConfig() throws URISyntaxException, LibraryFactoryException {
```

Comment:
`testAppEngineApiLibraryConfig` to be consistent? (Also, I think it should be `Api`, not `API`.)